container/types.proto

```diff
@@ -35,12 +35,33 @@ message Container {
   // empty. Containers with duplicated attribute names or attributes with empty
   // values will be considered invalid.
   //
-  // There are some "well-known" attributes affecting system behaviour:
+  // There are some "well-known" attributes affecting system behaviour.
+  //
+  // Attributes for sub networks:
   //
   // * __NEOFS__SUBNET \
   //   String ID of container's storage subnet. Container can be attached to
   //   only one subnet.
   //
+  // NeoFS provides native support of container ownership transfer with
+  // Non-Fungible Tokens (NFT). Requests signed by NFT owner considered as a
+  // requests from container owner by storage nodes. Inner ring nodes use NFT
+  // owner address to make settlements and allow NFT owner to change
+  // extended ACL rules.
+  // To enable ownership transfer, set these container attributes:
+  //
+  // * __NEOFS__NFT-ID \
+  //   String of token ID associated with the container
+  // * __NEOFS__NFT-Address \
+  //   Address of the contract that produced NFT, encoded in LittleEndian
+  // * __NEOFS__NFT-Type \
+  //   (Optional) Type of NFT. NeoFS supports only NEP-11 NFT (default value)
```

Comment:
Not clear, is it string `NEP-11`, or `NEP-11 NFT`?

Reply:
Added quotes for "NEP-11" type and "N3" chain.